src/helpers/generateBestSellerTitle.js

```diff
@@ -0,0 +1,7 @@
+import {bestSellingTv} from "/src/constants/inventory.js";
+
+function title() {
+    return bestSellingTv.brand + " " + bestSellingTv.type + " - " + bestSellingTv.name;
```

Comment:
Vind ik zelf altijd duidelijk te lezen en aan te passen
```suggestion
    return `${bestSellingTv.brand} ${bestSellingTv.type} - ${bestSellingTv.name}`;
```